test/unit/SourceLocator/Ast/FindReflectionsInTreeTest.php

```diff
@@ -213,6 +213,38 @@ public function testInvokeCallsReflectNodesForConstantByDefine(): void
 
         $mockReflection = $this->createMock(ReflectionConstant::class);
 
+        $strategy->expects($this->once())
```

Comment:
Not important, but `once()` is a static constructor

Reply:
I've just copy&pasted it :) 

Will do a special PR with these "fixes"

Reply:
No need for follow-up, was just fyi/future patches 😁